examples/libcloudforensics.py

```diff
@@ -90,6 +90,9 @@ def AddParser(provider, provider_parser, func, func_helper, args=None):
             'List EBS volumes in AWS account.')
   AddParser('aws', aws_subparsers, 'copydisk', 'Create an AWS volume copy.',
             args=[
+                ('--dst_zone', 'The AWS zone in which to copy the volume. By '
+                               'default this is the same as \'zone\'.',
```

Comment:
FYI - you can use double quotes instead of escaping the single quotes. If you really want single quotes in your text, then it's OK to use double quotes for your string:

```python
"This bug has been 'fixed'"
'This bug has been "fixed"'
```